example/package.json

```diff
@@ -18,7 +18,7 @@
     "@angular/platform-browser": "~12.0.0",
```

Comment:
The Angular versions should also be upgraded in the example by running `ng upgrade` here. Perhaps we can do this in a separate PR as well.

Reply:
I will update this after the final testing. I left like this in purpose for now. Thanks @jonkoops!